queries/inko/textobjects.scm

```diff
@@ -0,0 +1,262 @@
+; Classes
+(class) @class.outer
+
+(class
+  body: (class_body
+    .
+    "{"
+    .
+    (_) @_start @_end
+    (_)? @_end
+    .
+    "}"
+    (#make-range! "class.inner" @_start @_end)))
+
+; Traits
+(trait) @class.outer
+
+(trait
+  body: (trait_body
+    .
+    "{"
+    .
+    (_) @_start @_end
+    (_)? @_end
+    .
+    "}"
+    (#make-range! "class.inner" @_start @_end)))
+
+; Implementations
+(implement_trait) @class.outer
+
+(implement_trait
+  body: (implement_trait_body
+    .
+    "{"
+    .
+    (_) @_start @_end
+    (_)? @_end
+    .
+    "}"
+    (#make-range! "class.inner" @_start @_end)))
+
+(reopen_class) @class.outer
+
+(reopen_class
+  body: (reopen_class_body
+    .
+    "{"
+    .
+    (_) @_start @_end
```

Comment:
Would you mind removing the `@_end` captures from the `@_start` captures? Sorry this is something we are going to remove across the repo soon